lib/ZnapZend/ZFS.pm

```diff
@@ -12,7 +12,7 @@ use Data::Dumper;
 ### For "usual" datasets (filesystem,volume) `zfs` returns the
 ### properties inherited from higher level datasets; but for
 ### snapshots it only returns the same - not from higher snaps.
-my %inheritLevels = (
+our %inheritLevels = (
```

Comment:
note that the numbers in my example #516 was that you could turn them on or off by writing one or zero as a value ...

Reply:
I am still puzzled about proper (and type/name safe) passing of such "inherit" values as arguments to the function which can then be inspected. Looked hard at Class::struct to define and pass new() instances of that and checking whether this or that class member is true or false... would that be the way?

Reply:
with our you make this information accessible from the outside (%ZnapZend::ZFS::inheritLevels) ... note also that it is a Class variable and not an instance variable ... is this what you want?

Reply:
Well, in the current use effectively as an enum of distinct values to compare numeric inherit arguments/variables against, I don't think the nuance matters much. At least until someone decides to change its values on the fly :)
I guess a separate struct is cleaner for such purpose, use and concept alike.